packages/components/src/internal/components/lineage/node/LineageDetail.tsx

```diff
@@ -1,22 +1,88 @@
 import React, { FC, memo, useMemo } from 'react';
 import { Experiment, Filter } from '@labkey/api';
+import { Map } from 'immutable';
 
-import { DetailPanelWithModel, QueryConfig, SchemaQuery } from '../../../..';
+import { DetailPanel, LoadingSpinner, QueryColumn, resolveDetailRenderer, SchemaQuery } from '../../../..';
+
+import { InjectedQueryModels, QueryConfigMap, withQueryModels } from '../../../../public/QueryModel/withQueryModels';
+
+const ADDITIONAL_DETAIL_FIELDS = ['properties'];
 
 export interface LineageDetailProps {
     item: Experiment.LineageItemBase;
 }
 
+const LineageDetailImpl: FC<LineageDetailProps & InjectedQueryModels> = memo(props => {
+    const { actions, queryModels } = props;
+    if (queryModels.model.isLoading) return <LoadingSpinner />;
```

Comment:
Unfortunately, you'll need to handle all the problems here. Something like:

```tsx
if (queryModels.model.isLoading) return <LoadingSpinner />;
if (queryModels.model.hasLoadErrors) return <Alert>queryModels.model.loadErrors[0]</Alert>;
```

Reply:
Thanks. Done.